src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
 enchantment.openblocks.flimflam=Blödsinn
 
 openblocks.gui.blocks=Blöcke
-openblocks.gui.items=Items
+openblocks.gui.items=Gegenstände
```

Comment:
See [this comment](https://github.com/mezz/JustEnoughItems/pull/1438#pullrequestreview-186946544).

Reply:
I know, we are speaking Denglish today.